pinot-tools/src/main/java/org/apache/pinot/tools/service/PinotServiceManager.java

```diff
@@ -210,18 +210,24 @@ public boolean stopPinotInstance(ServiceStartable instance) {
     }
   }
 
-  public void start() {
-    LOGGER.info("Registering service status handler");
-    ServiceStatus.setServiceStatusCallback(_instanceId, new PinotServiceManagerStatusCallback(this));
-
+  public void start(boolean bootstrapMode) {
     if (_port < 0) {
       LOGGER.info("Skip Starting Pinot Service Manager admin application");
     } else {
+      LOGGER.info("Registering service status handler");
+      ServiceStatus.setServiceStatusCallback(_instanceId, new PinotServiceManagerStatusCallback(this));
+
       LOGGER.info("Starting Pinot Service Manager admin application on port: {}", _port);
       _pinotServiceManagerAdminApplication = new PinotServiceManagerAdminApiApplication(this);
       _pinotServiceManagerAdminApplication.start(_port);
     }
-    _isStarted = true;
+    if (!bootstrapMode) {
+      _isStarted = true;
```

Comment:
Should the `_isStarted` tracked for each instance since we can have multiple services running under the same service manager

Reply:
we do have this _isStarted per instance and used in the nested ServiceStatus.